src/utilities.js

```diff
@@ -22,21 +22,21 @@ const isSenator = (legislator) => {
   return legislator.email.includes('masenate')
 }
 
-function consolidateBillNumbers(arr) {
+function consolidateBills(bills) {
   const nameMap = new Map()
-  for (const obj of arr) {
-    const name = obj.shorthand_title.toLowerCase().trim()
-    const bill_number = obj.bill_number
+  for (const [bill_number, bill] of bills) {
+    const name = bill.shorthand_title.toLowerCase().trim()
+    chamber = Array.from(bill_number)[0] == 'H' ? 'house' : 'senate'
     if (nameMap.has(name)) {
-      // Found a duplicate
-      const duplicateObj = nameMap.get(name)
-      if (!duplicateObj.otherNames) {
-        duplicateObj.otherNames = [duplicateObj.bill_number]
-      }
-      duplicateObj.otherNames.push(bill_number)
+      // Found a paired bill
+      const pairedBill = nameMap.get(name)
+      pairedBill[`${chamber}BillNumber`] = bill_number
+      pairedBill[`${chamber}Status`] = bill.status
     } else {
       // First occurrence of this name
-      nameMap.set(name, obj)
+      bill[`${chamber}BillNumber`] = bill_number
+      bill[`${chamber}Status`] = bill.status
+      nameMap.set(name, bill)
     }
   }
   return Array.from(nameMap.values())
```

Comment:
I changed the assumption here from "any number of bills may be duplicates" to "each house or senate bill may be paired with up to one bill from the other chamber" since that's sort of an unofficial rule the legislature seems to follow 100% of the time, and thus how @JonathanCohn has organized the data in the spreadsheet. @ascampos if you feel strongly that this change is a bad idea, let me know!

Reply:
Thanks!